lib/defaults/defaults.go

```diff
@@ -188,6 +188,10 @@ var (
 
 	// ReportingPeriod is a period for reports in logs
 	ReportingPeriod = 5 * time.Minute
+
+	// UpdateClusterConfigPeriod is how often the cached cluster config is
+	// updated within the Auth Server.
+	UpdateClusterConfigPeriod = 10 * time.Second
```

Comment:
if we're polling, can we re-use one of existing constants? for example our nodes ping also (similar concept - also related to cluster state), so it would be easier to keep the same TTL in mind if you're doing support/troubleshooting?